style.css

```diff
@@ -0,0 +1,68 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #151515;
+}
+
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    color: #d3d3d3;
+}
+
+/**
+// Header Styles
+**/
+
+header h1 {
+    font-size: 1.2em;
+    color: #d3d3d3;
+    top: 5;
+    left: 0;
+    padding-bottom: 0.5em;
+    padding-top: 0.5em
+    /* position: absolute; */
+}
+
+input.searchbar {
+    background-color: #f7f7f7;
+    position: absolute;
+    top: 7px;
+    right: 0;
+    padding-top: 0.2em;
+    padding-bottom: 0.2em;
+    display: flex;
+    align-items: center;
+}
```

Comment:
I'm not too sure about using position: absolute for this.
![image](https://user-images.githubusercontent.com/36385459/118741864-ec679480-b814-11eb-9a86-a7163b25974f.png)


![image](https://user-images.githubusercontent.com/36385459/118741833-d6f26a80-b814-11eb-9064-216e65f92024.png)

I believe you should be able to do this using flexbox if you chose to, since the whitespace around should be the same amount, the content (title & searchbar) should just be in the vertically centered.